components/log-viewer-webui/server/src/plugins/S3Manager.ts

```diff
@@ -63,14 +71,14 @@ class S3Manager {
  * Initializes a Fastify plugin, which decorates the application with an S3 manager at the
  * "s3Manager" property only when all plugin options are valid.
  */
-export default fastifyPlugin(async (app, options: {region: Nullable<string>}) => {
-    const {region} = options;
+export default fastifyPlugin(async (app, options: {region: Nullable<string>, profile: string}) => {
+    const {region, profile} = options;
     if (null === region) {
```

Comment:
@coderabbitai In a separate PR, we should add an `isEnabled` option and check that instead. create an issue to track this.